src/instructlab/utils.py

```diff
@@ -448,8 +468,13 @@ def read_taxonomy_file(file_path: str, yaml_rules: Optional[str] = None):
         tax_path = "->".join(taxonomy_path.parent.parts)
         task_description = contents.get("task_description")
         documents = contents.get("document")
+
         if documents:
-            documents = get_documents(source=documents)
+            tag_stripper = HTMLTagStripper()
+            documents = [
+                tag_stripper.strip_html_tags(doc)
+                for doc in get_documents(source=documents)
```

Comment:
I think any stripping should be in the get_documents method. Also note that the sdg repo has a copy of this method which will need similar treatment. In reality, this change should probably only be in the sdg repo. The only user in this repo is now `ilab taxonomy diff` which does not need to actually process the files.

Reply:
Thanks for the comment, moved to get_document. I'm not understand this 
```
 In reality, this change should probably only be in the sdg repo. The only user in this repo is now ilab taxonomy diff which does not need to actually process the files.
```
Because the command `ilab data generate` seems till go to utils.py in this repo. 
My version is `0.17.1.dev240`

Reply:
https://github.com/instructlab/instructlab/blob/b65cfc9d9a3ee4ae041c780c7b46228e35c8e22a/src/instructlab/data/generate.py#L164

The generate_data method is now in the instructlab.sdg package.

Reply:
Thanks @bjhargrave , just notice that my sdg version is not new. I'll create a ticket in that repo.

Reply:
Issue created 
Issue: https://github.com/instructlab/sdg/issues/79
PR: https://github.com/instructlab/sdg/pull/80